packages/next/src/server/app-render/segment-value-encoding.ts

```diff
@@ -30,7 +30,7 @@ export function encodeSegment(
   return encodedName as EncodedSegment
 }
 
-export const ROOT_SEGMENT_KEY = '/'
+export const ROOT_SEGMENT_KEY = ''
```

Comment:
Was this intentional?

Reply:
yeah ended up being simpler to add the leading slash before sending the request